test/02-basic-find-test.py

```diff
@@ -239,7 +239,7 @@ def test_empty(self):
         try:
             self.db.find({})
         except Exception, e:
-            assert e.response.status_code == 400
+            assert e.response.status_code == 200
```

Comment:
Since it's an all_docs request, I'd like to see we get all the results back. I think the there's 20 rows for that particular set of tests so just assert that.

Also, it would be good to have a use_index test to ensure we still get that error back when they specify use_index, here's an example:

https://github.com/apache/couchdb-mango/blob/master/test/05-index-selection-test.py#L60